src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
-body {
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+body,
+h2,
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+  margin: auto;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)